### PR TITLE
[Vectorization] Bezier enhance

### PR DESCRIPTION
**Pull request by ARYA-BHARDWAJ**
This is regarding issue no #1184 @ice0  please have a look

### CODE REVIEW

**Comment by AnishGG** [comment]
It will be better if you could add `[Vectorization]` in the beginning of this P.R. ^__^
It seems to be misleading without it.

**Comment by ice0** [comment]
@ankit-kumar-dwivedi can you please take a look?

File: synfig-studio/src/synfigapp/vectorizer/centerlinetostrokes.cpp [hunk 1]
```diff
@@ -84,76 +84,68 @@ etl::handle<synfig::Layer> BezierToOutline(studio::PointList segment)
...
-                bline_point_list.back().set_width(segment[segment_size-1][2]);
-                bline_point_list.back().set_tangent((segment[segment_size-1].to_2d()-segment[segment_size-2].to_2d())*2);
+    default:{ /*Odd : 1 2 3 , 3 4 5, 5 6 7, 7 8 9
+                  Even : 1 2 3 4, 4 5 6, 6 7 8 */
+      int num = 0, point = 0;
+      if (segment_size & 1)
+      {
+        size = 2;
+        flag = 1;
+        ;
```
Comment (ankit-kumar-dwivedi) [inline synfig-studio/src/synfigapp/vectorizer/centerlinetostrokes.cpp]: remove this semicolon

**Comment by ankit-kumar-dwivedi** [comment]
@ARYA-BHARDWAJ the build failed for me with same errors as in this travis build -> https://travis-ci.com/github/synfig/synfig/jobs/297844389
please make sure the scope of variables are defined properly.

**Comment by ARYA-BHARDWAJ** [comment]
@ankit-kumar-dwivedi  please have a look

**Comment by ankit-kumar-dwivedi** [comment]
@ARYA-BHARDWAJ I am getting different results after running vectorization (see below image)
**Your version**
changed

**Previous version**
not_changed

Image I used -
line-drawing-of-duck-simple

On further debugging I saw all your bline_point_list have size() == 3
However, in the previous version, it is varied. I suggest you test the feature thoroughly with 2-3 image for

- [ ]  number of bline points
- [ ]  their size
- [ ]  shape of image must not be distorted

**Comment by ARYA-BHARDWAJ** [comment]
@ankit-kumar-dwivedi  Have a look I think it should work now

**Comment by ARYA-BHARDWAJ** [comment]
@ankit-kumar-dwivedi  please have a look now I think it should work now

**Comment by ankit-kumar-dwivedi** [comment]
@ice0 we are good to go here!
Good work @ARYA-BHARDWAJ

**Comment by ice0** [comment]
@ARYA-BHARDWAJ Looks good to me. Please squash and rebase commits. And I will merge it.

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**